util/fault_injection_test_env.h

```diff
@@ -82,6 +82,34 @@ class TestWritableFile : public WritableFile {
   FaultInjectionTestEnv* env_;
 };
 
+class TestRandomAccessFile : public RandomAccessFile {
```

Comment:
We don't want to modify rocksdb (i.e. everything outside utilities/titandb) for Titan changes moving forward. Let's temporarily make a wrapper (using `EnvWrapper`) of `FaultInjectionTestEnv` with the `TestRandomAccessFile` change and put it under titandb directory.

You can also send a PR to upstream rocksdb for the `TestRandomAccessFile` change.